measurements/api/measurements.py

```diff
@@ -537,7 +539,12 @@ def list_measurements(
         fpq = fpq.filter(Fastpath.test_name == test_name)
     if input_:
         fpq = fpq.filter(Fastpath.input == input_)
-
+    if confirmed is not None:
+        fpq = fpq.filter(False)
```

Comment:
This means basically that we can't filter by confirmed on the fastpath, correct?

Maybe we should add a comment above this line stating that is what is going on here.

Reply:
Correct. I put a comment in the PR - the idea is to then add the confirmed column.